src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -82,6 +78,13 @@ public CheckMonitoredTrip(MonitoredTrip trip) {
         this.trip = trip;
     }
 
+    /**
+     * For testing purposes only!
+     */
+    public void setJourneyState(JourneyState journeyState) {
+        this.journeyState = journeyState;
+    }
+
```

Comment:
I'm not sure this is the PR to do this in, but I've been thinking that since we're exposing JourneyState (or some portion of it) in the getMonitoredTrip endpoint, it probably does make sense to just remove the collection and have the JourneyState object as a field on MonitoredTrip.

Reply:
Fixed in 6fb9ef205237507ad27eaab1306f5bfa7e9284a4